docs/mockups/mottainai.html

```
@@ -0,0 +1,602 @@
+          </div>
+        </div>
+      </main>
+    </div>
+    <div class="backdrop" id="drawerBackdrop"></div>
+    <aside class="drawer" id="drawer">
+      <div class="drawer-head">
+        <div>
+          <div class="eyebrow">RECOVERY INSPECTOR</div>
+          <b id="drawerTitle">#378 Cleanup blocked</b>
+        </div>
+        <button class="icon-btn" data-close>ESC</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)